utils/math_utils.py

```diff
@@ -0,0 +1,12 @@
+import math
+
+
+def solve_quiz(alert_text):
```

Comment:
```suggestion
def solve_quiz(value):
     return str(math.log(abs((12 * math.sin(float(value))))))
```
If you implment utility function it should not be related on any web elements( alert messages)